README.md

```diff
@@ -10,26 +10,50 @@
 
 Determine in realtime what's happening inside your node process from the terminal. No need to instrument code to get the deets. Also splits stderr/stdout to help spot errors sooner.
 
+## Getting Started
+
 ### Install
+The preferred method is global install but can optionally be placed locally as well.
+
+`npm install -g nodejs-dashboard`
 
-`npm install nodejs-dashboard --save-dev`
 
 #### Add the reporting module to your code
 
-From within your `index.js` or app entry point simply require the `nodejs-dashboard` module.
+There are a couple patterns for including `nodejs-dashboard` in your code. Since this module isn't designed for production use it's best to conditionally include.
+
+##### Including via preload argument
+
+This pattern requires you to use a run-time flag (see below).
+
+`nodejs-dashboard -- node -r nodejs-dashboard index.js`
+
+##### Including via code
+
+From within a `dev.index.js` script or other dev entry point simply require the `nodejs-dashboard` module.
 
 ```
 require("nodejs-dashboard");
 ```
 
 #### Update your package.json
 
-It's recommended that you create a npm script to launch the dashboard.
+It's recommended that you create a npm script to launch the dashboard. How you decide to include the module will dictate the run script.
+
+For example, if you want to conditionally include via requires flag see the following example.
+```
+...
+"scripts": {
+    "dev": "nodejs-dashboard -- node -r nodejs-dashboard index.js"
+  }
```

Comment:
GLOBAL NIT: `"scripts"` and closing brace `}` throughout this whole doc seem misaligned.